src/main/java/com/minecolonies/coremod/inventory/InventoryCitizen.java

```diff
@@ -53,10 +54,12 @@
      */
     private final        NonNullList<ItemStack> offHandInventory = NonNullList.<ItemStack>withSize(1, ItemStackUtils.EMPTY);
     private final List<NonNullList<ItemStack>> allInventories;
+
     /**
-     * The index of the currently held item (0-8).
+     * The index of the currently held items (0-8).
      */
-    public        int                          currentItem;
+    public        int mainItem;
+    public        int offhandItem;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make offhandItem a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)